conanfile.txt

```diff
@@ -1,22 +1,19 @@
 [requires]
-boost/1.71.0
+boost/1.83.0
```

Comment:
boost 1.71 can't be used anymore on newer osx versions, it uses functions which were removed in c++17